content/community/articles.md

```diff
@@ -1,16 +1,16 @@
 ---
 id: articles
-title: Articles
+title: Artigos
 layout: community
 sectionid: community
 permalink: community/articles.html
 ---
 
-- [React How-to](https://github.com/petehunt/react-howto) - Pete Hunt's guide to the React ecosystem.
-- [9 things every React.js beginner should know](https://camjackson.net/post/9-things-every-reactjs-beginner-should-know) - Cam Jackson's guide for beginners.
-- [React "Aha" Moments](https://tylermcginnis.com/react-aha-moments/) - Tyler McGinnis' article on his collection of "Aha" moments with React.
-- [You're missing the point of React](https://medium.com/@dan_abramov/youre-missing-the-point-of-react-a20e34a51e1a) - Dan Abramov's article about the best parts of React.
-- [Timeline for Learning React](https://daveceddia.com/timeline-for-learning-react/) - Dave Ceddia's reccommended timeline for learning React and the React ecosystem.
-- [Simple React Development in 2017](https://hackernoon.com/simple-react-development-in-2017-113bd563691f) - Joshua Comeau's guide to showcase how easy it can be to start modern React development.
-- [React FAQ](https://reactfaq.site/) - An external site with articles that try to answer frequently asked questions about React.
-- [Visual Guide to State in React](https://daveceddia.com/visual-guide-to-state-in-react/) - Dave Ceddia's visual guide to React state.
+- [React How-to](https://github.com/petehunt/react-howto) - O Guia de Pete Hunt's para o ecossistema React.
```

Comment:
Opa, aqui precisa traduzir o que esta entre colchetes também (e.g React How-To, You're missing the point of React...)